package/libsolv.spec.in

```diff
@@ -36,8 +36,12 @@
 %endif
 # we need at least swig 1.3.40 for the bindings ($typemap support)
 %if 0%{?suse_version} != 1110
-%bcond_without python3
+%if 0%{?suse_version} < 1500
```

Comment:
btw this condition is slightly wrong as it will evaluate to true in case of fedora/rhel (not that it matters as those have their own spec)…

Reply:
FIxed.